### ISSUE TITLE
Latex expressions are not rendered when set to Axis#xticks

### ISSUE THREAD

**Issue by whatsthecraic** [MakieOrg/Makie.jl#2319]
As reported in https://discourse.julialang.org/t/makie-latex-in-tick-labels/86980

Given the following snippet:
```
ax = Axis(Figure()[1,1])
xlims!(ax, 1, nothing)
ax.xscale = log2
ax.xticks = ([1, 8, 32], [L"2^0", L"2^3", L"2^5"])
values = [1,2,3,4,5,6]
points = [1,2,4,8,16,32]
lines!(ax, points, values)
current_figure()
```

It is rendered by CairoMakie as:

![image](https://user-images.githubusercontent.com/11631433/194429838-57d37df4-9e08-454a-b721-8f44c11b3ff7.png)

**Comment by lazarusA** [comment]
currently, you need to define these kind of options at the initialisation step. The following works

``Julia
using GLMakie
fig = Figure(;font = "CMU Serif", fontsize=24)
ax = Axis(fig[1,1]; xscale=log10,
    xticks =([2^0, 2^3, 2^5], [L"2^0", L"2^3", L"2^5"]),
    xlabelsize = 22)
scatter!(ax, [2^0, 2^3, 2^5], 5*[2^0, 2^3, 2^5])
fig
```

![latexticks](https://user-images.githubusercontent.com/19525261/194478177-9b84a5d0-1f48-4782-82fe-6249a7641802.png)

**Comment by fatteneder** [comment]
I already proposed a fix for this: #2262

**Comment by jkrumbiegel** [comment]
This will be the next thing I look at, has been sitting around for too long already

**Comment by fatteneder** [comment]
fixed by #2322
